firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/GenerativeModel.kt

```diff
@@ -99,9 +99,10 @@ internal constructor(
 
             if (token.error != null) {
               Log.w(TAG, "Error obtaining AppCheck token", token.error)
-            } else {
-              headers["X-Firebase-AppCheck"] = token.token
             }
+            // The appcheck backend can differentiate between apps without appcheck, and wrongly
```

Comment:
```suggestion
            // The Firebase App Check backend can differentiate between apps without App Check enabled, and wrongly
```

Reply:
Is adding "enabled" accurate?

Reply:
Not really, since enabled implies that it's available but not used. The scenario I refer to here is where the app does not depend on a real implementation of AppCheck. In those cases, the app will not include any token at all, not even the error/dummy one.

Reply:
let's apply my other suggested edit :-)